backend/models/talk.js

```diff
@@ -0,0 +1,38 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var ratingSchema = new Schema({
+  rating: {
+    type: Number,
+    min: -1,
+    max: 1,
+    required: true
+  },
+  ratedBy: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  }
+});
+
+var talkSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true
```

Comment:
As the system is simple, there are not many fields for a talk. I decided to use **title** as an identifier for each talk so that users would not get confused from reading a simple list of talks.